src/utilities/Button.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled, { css } from "styled-components/macro";
+import { Link } from "react-router-dom";
+
+const Button = (props) => {
+  const handleClick = () => {
+    console.log(props);
+  };
+  return (
+    <BtnLink to={props.to}>
```

Comment:
- Ensure that component accomodates other use cases
e.g. black color, onclick, and specifics for each page

Reply:
Also remove consoe.log() code line